product-service/data/libs/constants.ts

```diff
@@ -0,0 +1 @@
+export const YOUR_EMAIL = "" || "kalet1984@gmail.com";
```

Comment:
What this supposed...? :) It's absolutely same as  
 `export const YOUR_EMAIL = "kalet1984@gmail.com";`  
Data of this type recommend to hold in env vars!